instruqt-tracks/sentinel-for-terraform-v4/04-restrict-domains-v1/assignment.md

```diff
@@ -0,0 +1,139 @@
+---
+slug: restrict-domains-v1
+id: boovju3egk3t
```

Comment:
Let's clear the `id:` fields out of `assignment.md` while we're moving to v3 of the track config